create_framework/index.rst

```diff
@@ -1,4 +1,4 @@
-Create your PHP Framework
+Create your own PHP Framework
 =========================
```

Comment:
Minor issue: in Symfony docs, the second line of the headline must be of the same length of the first line.